README.md

```diff
@@ -1,20 +1,29 @@
 ## Cloud SQL Connector for Java
 
-The Cloud SQL Connector for Java is a library for the MySQL/PostgreSQL JDBC and R2DBC drivers that allows a user 
-with the appropriate permissions to connect to a Cloud SQL database without having to deal with IP 
-whitelisting or SSL certificates manually.
+![CI Java 8][ci-badge-java-8]
```

Comment:
Is there a link to the continuous build available to the public? It might be nice to add it here if so. Ditto with the line below.

Reply:
I'm not sure that there is. The only one I'm aware of is the one on Fusion.